### ISSUE TITLE
what's the meaning of the average score in rank:pairwise objective?

### ISSUE THREAD

**Issue by chloe-wang** [dmlc/xgboost#5402]
I observed that the average value of the predicted scores is around 0.5. Because rank:pairwise only considers the relavant value between two samples in loss function, how can we get the average value of the predicted scores?

**Comment by trivialfis** [comment]
@chloe-wang Could you please be more specific.  From the context I'm not sure what you are trying to ask.  Maybe an example will help?

**Comment by chloe-wang** [comment]
@trivialfis thanks for the reply. From the perspective of lambdarank algo, the absolute scores of samples seem not important, since we only care about the relavant scores between two samples (if I'm understanding it correctly). So I'm wondering how can we get the absolute score when we are doing the prediction. Below is the distribution of the prediction scores for your reference.

![image](https://user-images.githubusercontent.com/22490931/76972460-96c1fe80-6969-11ea-9af7-0e2e0ebf1558.png)

**Comment by hcho3** [comment]
Running the predict() function will give you absolute scores, from which you should deduce the relative ordering between items in each query group.
